e2e-tests/gef/cypress/integration/scenarios/maker-should-not-submit-its-own-deals.spec.js

```diff
@@ -85,3 +91,352 @@ context('Submit application to UKEF', () => {
+        })
+        .then(({ body }) => {
+          body.items.forEach((item) => {
+            dealIds.push(item._id);
+          });
+        });
+    });
+
+    it('does not allow a maker/checker to submit own edited deals', () => {
+      // login as a maker and submit
+      cy.login(CREDENTIALS.MAKER);
+      cy.visit(relative(`/gef/application-details/${dealIds[2]}`));
+
+      // Make the deal an Automatic Inclusion Application
+      applicationDetails.automaticCoverDetailsLink().click();
+      automaticCover.automaticCoverTerm().each(($el, index) => {
+        $el.find('[data-cy="automatic-cover-true"]').trigger('click');
+        if (index === 7) {
+          $el.find('[data-cy="automatic-cover-false"]').trigger('click');
+        }
+      });
+      automaticCover.continueButton().click();
+      cy.url().should('eq', relative(`/gef/application-details/${dealIds[2]}/ineligible-automatic-cover`));
+      automaticCover.continueButton().click();
+      cy.url().should('eq', relative(`/gef/application-details/${dealIds[2]}/supporting-information/manual-inclusion-questionnaire`));
+      cy.uploadFile('upload-file-valid.doc', `${manualInclusion.url(dealIds[2])}/upload`);
+      manualInclusion.uploadSuccess('upload-file-valid.doc');
+      securityDetails.visit(dealIds[2]);
+      securityDetails.exporterSecurity().type('test');
+      securityDetails.facilitySecurity().type('test2');
+      securityDetails.continueButton().click();
+
+      // login as maker_checker only to delete a file.  file readded as maker
+      cy.login(CREDENTIALS.MAKER_CHECKER);
+      cy.visit(relative(`/gef/application-details/${dealIds[2]}`));
+
+      uploadFIles.supportingInfoManualInclusionButton().click();
+      uploadFIles.deleteSupportingDocument('upload-file-valid.doc').click();
+
+      cy.login(CREDENTIALS.MAKER);
+
+      cy.visit(relative(`/gef/application-details/${dealIds[2]}/supporting-information/manual-inclusion-questionnaire`));
+
+      cy.uploadFile('upload-file-valid.doc', `${manualInclusion.url(dealIds[2])}/upload`);
+      manualInclusion.uploadSuccess('upload-file-valid.doc');
+
+      cy.visit(relative(`/gef/application-details/${dealIds[2]}`));
+      // submit the deal
+      applicationDetails.submitButton().click();
+      applicationSubmission.commentsField().type('DTFS2-4698 Comments from original maker');
+      applicationSubmission.submitButton().click();
+      applicationSubmission.confirmationPanelTitle();
+
+      // login as a maker_checker and ensure that cannot return or submit to ukef
+      cy.login(CREDENTIALS.MAKER_CHECKER);
+      cy.visit(relative(`/gef/application-details/${dealIds[2]}`));
+      applicationPreview.returnButton().should('not.exist');
+      returnToMaker.submitButton().should('not.exist');
+    });
+  });
+
+  describe('BANK1_MAKER1 makes application, MakerChecker adds security questions only, MakerChecker should not be able to submit to ukef', () => {
```

Comment:
Can we be consistent with the wording for maker and maker-checker, lots of different variants

-BANK1_MAKER1
- MAKER
- MAKER_CHECKER
- MakerChecker
- maker/checker

Reply:
done